doc/contribute.rst

```diff
@@ -21,5 +21,20 @@ Tests can be written with `pytest <https://docs.pytest.org/en/latest/>`_
 or the `unittest <https://docs.python.org/3/library/unittest.html>`_ module.
 
 
```

Comment:
Is contribute.rst recognized by github? Otherwise change to CONTRIBUTING.md to [show](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) this information in pull requests?